### PR TITLE
make instructions gif in share audio accessible and provide equivilant translation

### PR DESCRIPTION
**Pull request by nurjinjafar**
in share audio dialog there is the introductary gif which shows the user what to do in order to ba able to share his/her audio. 
However, this information is not available to users with "sight deficiency", therefore. we aimed to make the img accessible by making it tapable and readable by screen readers in both english / german

### CODE REVIEW

File: lang/main.json
Comment (saghul) [inline lang/main.json:424]: Please don't mention the broser name. There are other browsers using the Chromium engine such as Edge or Brave.

**Review by saghul — APPROVED** [review]

**Comment by nurjinjafar** [comment]
hi @saghul, i saw you approved the PR, is there anything needs to be checked before it is merged?

**Comment by saghul** [comment]
I was waiting for the tests, merged now!

**Pull request merged**
